src/haddock/libs/libcns.py

```diff
@@ -117,42 +126,42 @@ def write_eval_line(param, value, eval_line='eval (${}={})'):
         return eval_line.format(param, value)
 
     else:
-        emsg = f'Unexpected type when writing CNS header: {type(value)}'
+        emsg = f"Unexpected type when writing CNS header: {type(value)}"
         log.error(emsg)
         raise TypeError(emsg)
 
 
 def load_ff_parameters(forcefield_parameters):
     """Add force-field specific parameters to its appropriate places."""
-    ff_param_header = f'{linesep}! FF parameters{linesep}'
-    ff_param_header += f'parameter{linesep}'
-    ff_param_header += f'  @@{forcefield_parameters}{linesep}'
-    ff_param_header += f'end{linesep}'
+    ff_param_header = f"{linesep}! FF parameters{linesep}"
+    ff_param_header += f"parameter{linesep}"
```

Comment:
Note that there is no need anymore to write the reading of `haddock.top` and `haddock.param` to the CNS input files. This is now handled by the `read_param.cns` script in each module

Reply:
removed!